### PR TITLE
Rename assets to RPG2k3 English RTP, add missing sounds

### PR DESCRIPTION
**Pull request by Ghabry**
-

### CODE REVIEW

**Comment by jetrotal** [comment]
We may also need to review the attribution part of the authors file, to include the proper credits to it  
  
  
https://community.easyrpg.org/t/basis-png-by-jikmok/146

**Comment by Ghabry** [comment]
Yeah the attribution is kinda messy and in some cases so detailed that it does not fit into our table.

There could be also a "FILE_NAME.license" file next to each asset. The Editor could process this file and display copyright information in the asset browser.

Was also thinking of embedding Exif chunks (supported since 2017 for PNG) into the images for attribution. Sure they are often lost due to applications not writing them but better than nothing ^^

**Comment by jetrotal** [comment]
[https://github.com/MedicineStorm/RTP/blob/master/AUTHORS.md](https://github.com/MedicineStorm/RTP/blob/master/AUTHORS.md)  
  
Here, Medicine Storm already researched and organized some of the OGA links

**Review by jetrotal — APPROVED** [review]

File: AUTHORS.md [hunk 1]
```diff
@@ -7,63 +7,71 @@ License URL: https://creativecommons.org/licenses/by/4.0/
...
+System/SystemC.png          | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
+System2/System2a.png        | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
+System2/System2b.png        | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
+System2/System2c.png        | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
 Title/Title1.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
 Title/Title2.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
 Title/Title3.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
 Title/Title4.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
+
+[CC0]: https://creativecommons.org/publicdomain/zero/1.0/deed.de
```
Comment (carstene1ns) [inline AUTHORS.md:75]: `.en` would have been better.

Comment (Ghabry) [reply AUTHORS.md:75]: Automatic redirects :/
Good catch fixing this the next time I make a PR here.

**Pull request merged**